### DISCUSSION TITLE
After climbing, deep sound starts since update to built": "2022-01-23T05:07:32+00:00"

### DISCUSSION THREAD

**Discussion by Adi-18** [flybywiresim/aircraft#6579] [Help] [answered]
Since  "built": "2022-01-23T05:07:32+00:00",

After preparing Flight , take off and climb over 5000(?) ft, there starts a deeb sound monotonously (about 60 Hz or so).

There is no warning lights or corresponding messages in ECAM and its horrible until the end of the flight.

What may it be?

**Comment by CptBayer** [comment, marked as answer]
Hi,

altitude alert ..... press button on the glare .... (left side, button on top :-P )

![grafik](https://user-images.githubusercontent.com/76531893/151237581-45240ca8-d2ee-4a43-898d-a798cdf6cbd4.png)

greets ..

**Reply by Adi-18** [reply]
Hi, at the time I wrote this, there  was e new feature in develop build. But there was a bug I could not turn of altitude alert. View days later it was fixed in new develop build and now it works as desired.
